src/test/java/org/gridsuite/sensitivityanalysis/server/dto/SensitivityAnalysisInputDataTest.java

```diff
@@ -127,7 +127,7 @@ public void testEmptyInputTranslation() {
             .build();
         ReporterModel reporter = new ReporterModel("a", "b");
         SensitivityAnalysisRunContext context;
-        context = new SensitivityAnalysisRunContext(NETWORK_UUID, VARIANT_ID, inputData, null, null, null, null);
+        context = new SensitivityAnalysisRunContext(NETWORK_UUID, VARIANT_ID, null, null, null, "OpenLoadFlow", inputData);
```

Comment:
extract "OpenLoadFlow" in const variable

Reply:
Ok done.